src/TConsole.cpp

```diff
@@ -2187,6 +2205,14 @@ TConsole* TConsole::createBuffer(const QString& name)
 
 void TConsole::resetMainConsole()
 {
+//resetProfile should reset also UserWindows
+    QMutableMapIterator<QString, TDockWidget*> itDockWidget(mDockWidgetMap);
+    while (itDockWidget.hasNext()) {
+        itDockWidget.next();
+        itDockWidget.value()->close();
+        itDockWidget.remove();
```

Comment:
After I used `resetProfile()`, the userwindow popped back inside Mudlet. I dragged it out, but it was no longer resizable. Do you get this?

Reply:
Yes, has to do with the restore Layout option if you create UserWindow by using openUserWindow(windowName, [restoreLayout]) and restoreLayout has the value false then every issue is gone. Don't know the reason though.
Had the same issue also in the actual version of Mudlet sometimes but I don't know which causes it and it was difficult to reproduce.